tests/ctl/core/test_api.py

```diff
@@ -654,6 +659,14 @@ async def test_system_create(
             system.data_security_practices
             == "We encrypt all your data in transit and at rest"
         )
+        assert system.cookie_max_age_seconds == 31536000
+        assert system.uses_cookies == True
+        assert system.cookie_refresh == True
+        assert system.uses_non_cookie_access == True
```

Comment:
minor, in Python, you'd typically want to do the comparison like: `system.uses_cookies is True` for a boolean, like you did below with `is None` but only worth changing if you're pushing another commit for CI

Reply:
gosh i don't know why i always struggle to get this right 😄 i'll update, it's sloppy - this is what happens when developing in a rush. thanks for catching 👍